python/lsst/afw/cameraGeom/detector.cc

```diff
@@ -61,15 +60,15 @@ PYBIND11_PLUGIN(_detector) {
     /* Constructors */
     cls.def(py::init<std::string const &, int, DetectorType, std::string const &, geom::Box2I const &,
                      table::AmpInfoCatalog const &, Orientation const &, geom::Extent2D const &,
-                     CameraTransformMap::Transforms const &>(),
+                     TransformMap::Transforms const &>(),
             "name"_a, "id"_a, "type"_a, "serial"_a, "bbox"_a, "ampInfoCatalog"_a, "orientation"_a,
             "pixelSize"_a, "transforms"_a);
 
     /* Operators */
     cls.def("__getitem__",
-            (std::shared_ptr<table::AmpInfoRecord const>(Detector::*)(int) const) & Detector::_get, "i"_a);
+            (std::shared_ptr<table::AmpInfoRecord const> (Detector::*)(int) const) & Detector::_get, "i"_a);
```

Comment:
Lots of (probably clang-format) spurious whitespace changes on this commit in this file.  May be past the point where it's worth separating them, though.

Reply:
I'm afraid so.